components/src/dynamo/vllm/omni/omni_handler.py

```
@@ -337,14 +308,48 @@ def _engine_inputs_from_video(self, req: NvCreateVideoRequest) -> EngineInputs:
+            if response_format == "url":
+                storage_path = f"images/{request_id}/{uuid.uuid4()}.png"
+                url = await upload_to_fs(
+                    self.media_fs, storage_path, image_bytes, self.media_base_url
+                )
+                outlist.append(url)
+            elif response_format == "b64_json" or response_format is None:
+                img_base64 = base64.b64encode(image_bytes).decode("utf-8")
+                data_url = f"data:image/png;base64,{img_base64}"
+                outlist.append(data_url)
+            else:
+                raise ValueError(f"Invalid response format: {response_format}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)